common/api/ui-framework.api.md

```diff
@@ -3041,7 +3043,7 @@ export interface IModelInfo {
     // (undocumented)
     name: string;
     // (undocumented)
-    projectInfo: ProjectInfo;
+    projectInfo: ITwin;

```

Comment:
Is this name accurate? When I see "iTwin" I think of an iModel with attached metadata, but this seems to represent a project rather than a single iModel. Also, do we want to call it "iTwinId" instead of "contextId"?

Reply:
The name iTwin was an early choice to encompass the greater set of projects, assets, or custom contexts. I would have to defer to @calebmshafer for whether or not that fits the business terminology.
As for the name "contextId" I agree, but there was some thought to roll that kind of semantic renaming of variables and functions into another PR. That is something I will do.

Reply:
I just find this name very confusing. Perhaps it was supposed to be "iTwin Context" or something else more detailed than just "iTwin"?
@calebmshafer if this is the intended terminology, I guess I don't know what an iTwin is anymore? 😅

Either way, I think renaming variables in a separate PR would just add to the confusion, not to mention it will require you to get everyone's approval to merge again.